NewsApi/auth.py

```diff
@@ -1,53 +1,44 @@
 import operator
 
```

Comment:
Is the code from this file used? there seem to be another login and register view in the routes.py